tests/test_utils.py

```diff
@@ -203,9 +207,11 @@ def read_vic_ascii(filepath, parse_dates=True, datetime_index=None, sep='\t',
 
     if parse_dates:
         time_cols = ['YEAR', 'MONTH', 'DAY']
-        df.index = pd.to_datetime(df[time_cols])
+        df.index = pd.to_datetime(df.YEAR * 10000 + df.MONTH * 100 +
+                                  df.DAY, format='%Y%m%d')
```

Comment:
Done. This was because of the pandas version issue I was having between `0.18.0` and `0.18.1` with the behavior of `pd.to_datetime`.